CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -138,17 +146,12 @@ public function buildQuickForm(): void {
       }
     }
     else {
-      // Dig deeper - this code is sublimely stupid.
```

Comment:
This is easily my favorite comment in our codebase. I'm sorry to see it go @totten

Reply:
Well, I guess that code had to wander 7 years through the wilderness, but now... it's sublimely smart... I hope...